docs/content/preview/secure/encryption-at-rest.md

```diff
@@ -131,3 +131,11 @@ You can disable cluster-wide encryption as follows:
     ```output
     Encryption status: DISABLED
     ```
+
+
+{{< note title="Note" >}}
```

Comment:
Far as I can tell, docs only describe in-place snapshot. Are you talking about restoring a snapshot on another cluster? Backup/restore pages don't mention EAR at all, but that seems like a more suitable spot, but there are no instructions on what/how to do this
https://docs.yugabyte.com/preview/manage/backup-restore/snapshot-ysql/

Reply:
@ddorian How about -
Store your keys securely, including those that have been rotated. If you want to restore distributed snapshots of an encrypted cluster to a new cluster, you need to copy the keys used for encryption (including rotated keys) to the new cluster. (Backup, restore, and key management are automated in YugabyteDB Anywhere.)